CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
   The `rover template` commands now include four more languages.
 
+- **Allow skip checking new versions of rover - @tsing, #1394**
+
+  The checking can be skipped by the global `--skip-update-check` flag.
+
```

Comment:
We've already released `0.9.1` and so this entry will be included in the _next_ release of Rover. For now you can just remove this from this PR and we'll add it in during our release process 👍🏼

Reply:
@EverlastingBugstopper I have force updated the commit to revert the changes in the `CHANGELOG.md`.